src/common/services/contracts.js

```diff
@@ -0,0 +1,21 @@
+import RelayerContract from 'culturestake-contracts/build/contracts/Relayer.json';
+import AdminContract from 'culturestake-contracts/build/contracts/Admin.json';
+import VoteContract from 'culturestake-contracts/build/contracts/Vote.json';
+
+import web3 from './web3';
```

Comment:
This can be changed to absolute paths as `import web3 from '~/common/services/web3';`